### PR TITLE
add images and game page and result page

### PR DESCRIPTION
**Pull request by KatiaAsh**
but it's not finished there still small bugs in CSS
relates #10 #6
 * Result page 
screenshot from 2017-10-02 19-11-12

 * Game page
screenshot from 2017-10-02 19-11-24

### CODE REVIEW

File: game.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  height: 5000;
+  box-sizing: border-box;
+  margin-bottom: 20px;
+  width: 5000;
+  position: fixed;
+}
+
+#gn-hide {
+  text-align: center;
+  display: none !important;
```
Comment (Wyrd00) [inline game.css]: not sure why !important is necessary here - consider removing?

File: game.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+#player-container {
+  z-index: 1000;
+  margin-right: 50%;
+  direction: ltr;
+  height: 5000;
+  box-sizing: border-box;
+  margin-bottom: 20px;
+  width: 5000;
+  position: fixed;
+}
```
Comment (Wyrd00) [inline game.css:22]: personal preference on my part, but changing this to class would really ease my OCD

File: game.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+#gn-hide {
```
Comment (Wyrd00) [inline game.css]: and this too 😆

**Review by Wyrd00 — CHANGES_REQUESTED** [review]

File: game.css (hunk 1 above)
Comment (KatiaAsh) [reply game.css]: :+1:

File: game.css (hunk 2 above)
Comment (KatiaAsh) [reply game.css:22]: :+1:

**Review by Wyrd00 — APPROVED** [review]

**Pull request merged**